tests/services/test_base.py

```diff
@@ -126,3 +127,63 @@ async def test_multiservice_stop_gracefully_stops_service_that_takes_too_long_to
     assert (
         not service_2.cancelled
     )  # we're not supposed to cancel it as it's already stopping
+
+
+config = {
+    "elasticsearch": {},
+    "service": {},
+}
+
+
+def test_parse_connectors_with_no_connectors():
+    local_config = deepcopy(config)
+    service = BaseService(local_config)
+    assert not service.connectors
+
+
+def test_parse_connectors():
```

Comment:
nit: there's no test here that has two different ids and checks that they're parsed correctly. I think it would be beneficial to have one.

Reply:
I think I will just update this tests to have two connectors.